src/app/interfaces/passwordState.interface.ts

```diff
@@ -0,0 +1,4 @@
+export interface PasswordState {
+    title: string;
```

Comment:
Please, check if we can make these props are readonly

Reply:
done